lambda/action_network.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import os
+from typing import Dict
+
+import requests
+from dateutil import parser
+
+log = logging.getLogger(__name__)
+
+
+def get_object_or_empty_dict(key: str, event: Dict) -> Dict:
+    location = event.get(key)
+    if not location:
+        location = {}
+    return location
+
+
+def make_location(event: Dict) -> str:
+    location = get_object_or_empty_dict('location', event)
+    if location.get('country', 'US') != 'US':
+        return '{city}, {country}'.format(
+            city=location.get('locality'), country=location.get('country'))
+    return location.get('postal_code', '')
+
+
+def get_event_name(event: Dict) -> str:
+    name = event.get('name', None)
+    if not name:
+        name = event.get('title', '')
+    return name
+
+
+def get_organizer(event: Dict) -> Dict:
```

Comment:
I really dislike one-line functions; just call it

Reply:
get_organizer may be a one_line function but there's a lot of complexity in there that I think is worth encapsulating.